daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/ex_nums.tdml

```diff
@@ -37,62 +37,69 @@
   </tdml:defineConfig>
 
   <tdml:parserTestCase
-    name="ex_nums_parse_runtime1"
     model="ex_nums.dfdl.xsd"
+    name="ex_nums_runtime1"
     config="config-runtime1">
     <tdml:document>
-      <tdml:documentPart type="file">ex_nums_parse.dat</tdml:documentPart>
+      <tdml:documentPart type="file">ex_nums.dat</tdml:documentPart>
     </tdml:document>
     <tdml:infoset>
-      <tdml:dfdlInfoset type="file">ex_nums_unparse_runtime1.xml</tdml:dfdlInfoset>
+      <tdml:dfdlInfoset type="file">ex_nums.dat.xml.runtime1</tdml:dfdlInfoset>
```

Comment:
Do the different runtimes have different infosets? Does this mean there is a bug somewhere, or is this a implementation detail that one of the runtimes just doesn't support something?

Reply:
Yes, runtime1 and runtime2 have different infosets because Java code and C code will format floating point numbers closely but not exactly the same as each other.  We would need to either make Java and C format floats exactly the same (very difficult) or make TDMLRunner compare floats semantically, not syntactically ([DAFFODIL-2402](https://issues.apache.org/jira/browse/DAFFODIL-2402)).